pkg/codec/vpx/vpx.go

```diff
@@ -306,6 +306,15 @@ func (e *encoder) ForceKeyFrame() error {
 func (e *encoder) Controller() codec.EncoderController {
 	return e
 }
+func (e *encoder) SetBitRate(bitRate int) error {
+	e.mu.Lock()
+	defer e.mu.Unlock()
+	e.cfg.rc_target_bitrate = C.uint(bitRate) / 1000
```

Comment:
should the input param be `bitRateKbps` to clarify the parameter?

Reply:
sure

Reply:
SetBitRate is actually an interface